### ISSUE TITLE
Lockscreen does not override incoming calls

### ISSUE THREAD

**Issue by jez9999** [Pi-Developers/Pi-Locker#3]
Even when the lockscreen is enabled, if you put the phone into sleep mode and then receive a call, the lockscreen is not displayed - the incoming call screen is displayed.  I want to unlock the phone *before* showing the incoming call screen.  This is still happening in version 5.0.3.

**Comment by Pi-Developers** [comment]
We will try seeing this, afaik the syatem disables any lockscreen at calls to improve functionality

**Comment by jez9999** [comment]
Yeah, but it doesn't improve functionality for me.  It makes it too easy to accidentally answer incoming calls.  Is there literally NO way to override this stupid incoming call screen?

**Comment by Pi-Developers** [comment]
I searched alot but sadly till now , a big no :(

**Comment by jez9999** [comment]
I'm sure it used to be possible.  I used to have a lockscreen I had to unlock to get at the incoming call, several versions of Android ago.

**Comment by jez9999** [comment]
What about modifying the callscreen itself?  That must be possible, and it could be turned into a lockscreen (or at least a screen where it is very difficult to accidentally answer the call).

**Comment by Pi-Developers** [comment]
this will need a root access
and we want app to be available for everyone even noobs

**Comment by jez9999** [comment]
Well how about releasing a version that does this with root access?  I have root access and damnit I am sick of accidentally answering calls because I lose my lockscreen. :-)

**Comment by Pi-Developers** [comment]
ok, we will discuss this, thank you for this suggestion, if I found another way i will tell you
